mat64/cholesky_example_test.go

```diff
@@ -0,0 +1,58 @@
+package mat64_test
+
+import (
+	"fmt"
+
+	"github.com/gonum/matrix/mat64"
+)
+
+func ExampleCholesky() {
+	a := mat64.NewSymDense(4, []float64{108, -21, -9, -96, -21, 7, 5, 15, -9, 5, 61, 25, -96, 15, 25, 142})
```

Comment:
Format the data over four lines and it would be nice to have a comment explaining that the lower triangle could be zeroed.